app/models/comment.rb

```diff
@@ -1,5 +1,5 @@
 class Comment < ApplicationRecord
-  validates :text_body
+  validate :text_body
```

Comment:
What does this do? Are we using custom validations?

Reply:
I kept getting this: `You need to supply at least one validation` I think it needs at least one validator following the line

Reply:
Ok sounds good, yeah, I can create a validation for that during the sprint